In [17]:
#3
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

#4
%matplotlib inline
#5
gdp_df=pd.read_csv('../data/gdp_percapita.csv')
gdp_df.head()

Country or Area  Year        Value  Value Footnotes
0     Afghanistan  2019  2065.036398              NaN
1     Afghanistan  2018  2033.779002              NaN
2     Afghanistan  2017  2058.383832              NaN
3     Afghanistan  2016  2057.062164              NaN
4     Afghanistan  2015  2068.265904              NaN

In [18]:
#6
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

internet_df=pd.read_csv('../data/internet_use.csv',nrows=4496)
internet_df.head()

Country or Area  Year     Value  Value Footnotes
0     Afghanistan  2014  6.390000              NaN
1     Afghanistan  2013  5.900000              NaN
2     Afghanistan  2012  5.454545              NaN
3     Afghanistan  2011  5.000000              NaN
4     Afghanistan  2010  4.000000              NaN

In [19]:
#6 Continued
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

gdp_df=pd.read_csv('../data/gdp_percapita.csv')
print(gdp_df.shape)

(6870, 4)


Answer: The shape of the gdp dataset is 6,870 rows and 4 columns

In [20]:
#6 Continued
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

internet_df=pd.read_csv('../data/internet_use.csv',nrows=4496)
print(internet_df.shape)

(4496, 4)


Answer: The shape of the internet use dataset is 4,496 rows and 4 columns

In [21]:
#6 Continued
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

gdp_df=pd.read_csv('../data/gdp_percapita.csv')
print(type ('Country or Area'))
print(type ('Year'))
print(type ('Value'))
print(type ('Value Footnotes'))     

<class 'str'>
<class 'str'>
<class 'str'>
<class 'str'>


In [22]:
#6 Continued
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

internet_df=pd.read_csv('../data/internet_use.csv',nrows=4496)
print(type ('Country or Area'))
print(type ('Year'))
print(type ('Value'))
print(type ('Value Footnotes'))     

<class 'str'>
<class 'str'>
<class 'str'>
<class 'str'>


In [23]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

gdp_df=pd.read_csv('../data/gdp_percapita.csv')
print(gdp_df.tail(10))

     Country or Area                                      Year        Value  \
6860        Zimbabwe                                      1997  3580.048793   
6861        Zimbabwe                                      1996  3520.430146   
6862        Zimbabwe                                      1995  3226.413930   
6863        Zimbabwe                                      1994  3263.934978   
6864        Zimbabwe                                      1993  3033.504852   
6865        Zimbabwe                                      1992  3054.889178   
6866        Zimbabwe                                      1991  3426.598094   
6867        Zimbabwe                                      1990  3324.348171   
6868   footnoteSeqID                                  Footnote          NaN   
6869               2  Excludes South Sudan after July 9, 2011.          NaN   

      Value Footnotes  
6860              NaN  
6861              NaN  
6862              NaN  
6863              NaN  
6864      

In [24]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

internet_df=pd.read_csv('../data/internet_use.csv',nrows=4496)
print(internet_df.tail(10))

     Country or Area      Year     Value  Value Footnotes
4486        Zimbabwe      2001  0.799846              NaN
4487        Zimbabwe      2000  0.401434              NaN
4488        Zimbabwe      1999  0.161676              NaN
4489        Zimbabwe      1998  0.081648              NaN
4490        Zimbabwe      1997  0.033080              NaN
4491        Zimbabwe      1996  0.016790              NaN
4492        Zimbabwe      1995  0.007684              NaN
4493        Zimbabwe      1994  0.001739              NaN
4494        Zimbabwe      1990  0.000000              NaN
4495   footnoteSeqID  Footnote       NaN              NaN


In [25]:
#10.
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

gdp_df=pd.read_csv('../data/gdp_percapita.csv')
gdp_trimmed_df=gdp_df.loc[:, ['Country or Area', 'Year', 'Value']]
print(gdp_trimmed_df.head())

  Country or Area  Year        Value
0     Afghanistan  2019  2065.036398
1     Afghanistan  2018  2033.779002
2     Afghanistan  2017  2058.383832
3     Afghanistan  2016  2057.062164
4     Afghanistan  2015  2068.265904


In [26]:
#10 Continued
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

internet_df=pd.read_csv('../data/internet_use.csv',nrows=4496)
internet_trimmed_df=internet_df.loc[:, ['Country or Area', 'Year', 'Value']]
print(internet_trimmed_df.head())

  Country or Area  Year     Value
0     Afghanistan  2014  6.390000
1     Afghanistan  2013  5.900000
2     Afghanistan  2012  5.454545
3     Afghanistan  2011  5.000000
4     Afghanistan  2010  4.000000
